ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -246,6 +246,9 @@ def _generate_wan_load_balance_policy(self, name: str, input_dict: dict, context
         wan_local_path_group_names = [path_group["name"] for path_group in self.shared_utils.wan_local_path_groups]
         wan_load_balance_policy = {"name": name, "path_groups": []}
 
+        # Handle constraints
+        wan_load_balance_policy |= get(input_dict, "constraints", default={})
```

Comment:
I have not come accross the "|=" bitwise operator before in AVD code, should we refactor like this?

```suggestion
        wan_load_balance_policy = wan_load_balance_policy | get(input_dict, "constraints", default={})
```

Reply:
I would prefer just `.update()` as we have everywhere. But in this case I would actually just insert the key in line 247 since suspect we remove `None` values somewhere later.